component/src/main/java/me/androidbox/component/agenda/AgendaDaySelector.kt

```diff
@@ -0,0 +1,84 @@
+package me.androidbox.component.agenda
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.R
+import me.androidbox.component.general.CalendarDayButton
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.Orange
+import java.time.ZonedDateTime
+
+@Composable
+fun AgendaDaySelector(
+    date: ZonedDateTime = ZonedDateTime.now(),
+    isSelectedDay: ZonedDateTime = ZonedDateTime.now(),
+    modifier: Modifier = Modifier,
+    onSelected: (ZonedDateTime) -> Unit,
+) {
+
+    Column(modifier = modifier) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceEvenly) {
+
+            /* Display only 6 days */
+            (0L..6L).forEach { day ->
+                CalendarDayButton(
+                    modifier = Modifier
+                        .clip(shape = RoundedCornerShape(100.dp))
+                        .background(color = Orange)
+                        .size(width = 40.dp, height = 60.dp),
+                    date = date.plusDays(day),
+                    isSelected = isSelectedDay.dayOfWeek == date.plusDays(day).dayOfWeek,
+                    onSelected = { date ->
+                        onSelected(date)
+                    }
+                )
+            }
+        }
+
+        Spacer(Modifier.height(34.dp))
+        val displayDate = if(isSelectedDay.dayOfWeek == date.dayOfWeek) stringResource(R.string.today) else "${date.dayOfMonth} ${date.month} ${date.year}"
```

Comment:
`isSelectedDay` sounds like a boolean, but it isn't. `selectedDate` would make more sense